src/Tribe/Views/V2/Views/By_Day_View.php

```diff
@@ -305,7 +306,8 @@ static function ( $event ) use ( $start, $end, $use_site_timezone, $site_timezon
 
 						return $event_localized_start_date->format( Dates::DBDATETIMEFORMAT ) <= $end
 						       && $event_localized_end_date->format( Dates::DBDATETIMEFORMAT ) >= $start;
-					} );
+					}
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: a90de19cda9eb0043207ce4616b3a4dc</sub>